src/main/java/com/avispl/symphony/dal/infrastructure/management/polycom/lens/PolyLensCommunicator.java

```diff
@@ -802,22 +800,86 @@ private void populateDeviceDetails() {
 	 * @return List<AggregatedDevice> aggregated device list
 	 */
 	private List<AggregatedDevice> cloneAndPopulateAggregatedDeviceList() {
-		List<AggregatedDevice> resultAggregatedDeviceList = new ArrayList<>();
 		synchronized (aggregatedDeviceList) {
-			for (AggregatedDevice aggregatedDevice : aggregatedDeviceList) {
+			for (AggregatedDevice aggregatedDevice : cachedAggregatedDeviceList) {
 				Map<String, String> stats = aggregatedDevice.getProperties();
 				List<AdvancedControllableProperty> controllableProperties = aggregatedDevice.getControllableProperties();
 				stats = mapMonitoringProperty(stats);
-				if (aggregatedDevice.getDeviceOnline()) {
+				if (Boolean.TRUE.equals(aggregatedDevice.getDeviceOnline())) {
```

Comment:
Is there any specific reason the check needs to be done like this? :)

Reply:
> Is there any specific reason the check needs to be done like this? :)

As far as I remember, controllable properties are only shown when the device is online :)